subprojects/android-test/ui-testing-core/build.gradle.kts

```diff
@@ -24,8 +25,10 @@ dependencies {
     api("androidx.appcompat:appcompat:$androidXVersion")
     api("androidx.recyclerview:recyclerview:$androidXVersion")
     api("com.google.android.material:material:$androidXVersion")
+    api("com.google.android.gms:play-services-base:17.0.0")
```

Comment:
why?

Reply:
It needs for PlayServices file. checkPlayServices is called from AbstractLaunchRule.
We already have had problems with dependency leak to inhouse from ui-testing-maps.
Now my goal is clear in-house module after we should refactor whole ui-testing-core

Reply:
it is better to have a mess in inhouse than ui-testing core

Reply:
I don't understand intention of your message. 
Where is a problem? We had playservices check in inhouse runner and didn't have direct dependency. In future we can discuss how we see inhouse runner.
But now i'm use principle if you use something u need it in your dependency.

Reply:
one could use ui testing core without inhouse runner, why core comes with play services dependency?

Reply:
because it needs PlayServices check.
I agree with that in future we should extract google deps to own module. 'ui-google-rule' for example. But now my focus on clearing inhouse runner.

Reply:
but ui testing core doesn't use play services as a dependency, you introduced it with this PR

Reply:
but inhouse runner don't use play services but has a dependency.
I can do new module if u want.